<h1>Information Exposure Through an Error Message</h1>
<p>The software generates an error message that includes sensitive
	information about its environment, users, or associated data. You need
	to use log4j.Logger to log your error instead.</p>
<p>You need to put a throw statement in the catch block and don't
	use log4j.Logger if you want.</p>
<h2>Noncompliant Code Example</h2>





<h3>Example 1</h3>
<p>In the following example, sensitive information might be printed
	depending on the exception that occurs.</p>
<pre>
try {
	//TO DO
}catch (Exception e) { //Please log error using org.apache.log4j.Logger library
	e.printStackTrace(); //Please remove printStackTrace()
}
</pre>
<p>If an exception related to SQL is handled by the catch, then the
	output might contain sensitive information such as SQL query structure
	or private information. If this output is redirected to a web user,
	this may represent a security problem.</p>

<h3>Example 2</h3>

<p>In the following example, You need to put a throw statement in
	the catch block.</p>
<pre>
public EventResponse perform(Event e) throws EventException {
		try {
			//TO DO
		}catch (Exception e) {//Please throw an Exception or remove throws
			//TO DO
		}
}
</pre>

<h2>Compliant Solution</h2>

<h3>Example 1</h3>
<pre>
try {
	//TO DO
}catch (Exception e) {
	log.error(e.getMessage()); // Log error using org.apache.log4j.Logger library
}
</pre>

<h3>Example 2</h3>
<pre>
public EventResponse perform(Event e) throws EventException {
		try {
			//TO DO
		}catch (Exception e) {
			throw new EventException(new ErrorHandler(e).getMessage(),e);
		}
}
</pre>

<p style="color: red">Note: If your class extends
	ServiceCommandSupport or BasicCommandSupport or DBDAOSupport. You
	already has log variable (name: log; type: log4j.Logger). Otherwise you
	can declare log variable by import org.apache.log4j.Logger library.</p>
