<h1>This rule detects invalid method comment</h1>
<h2>Noncompliant Code Example</h2>
<pre>
	public boolean isNameEmpty(String name) {
	  return name.length() == 0; //name could be null
	}
</pre>

<pre>
	public boolean isNameEmpty() {
		String name = null;//assign null value
		return name.length() == 0; // name: Please avoid NullPointerException
	}
</pre>

<pre>
	public boolean isNameEmpty(){
		String name;//don't assign value
		return name.length() == 0; // name: Please avoid NullPointerException
	}
</pre>

<h2>Compliant Solution</h2>
<pre>
	public boolean isNameEmpty() {
		String name = "Your name";
		return name.length() == 0;
	}
</pre>

<pre>
	public boolean isNameEmpty() throws NullPointerException {
		String name = null;
		return name.length() == 0;
	}
</pre>

<pre>
	public boolean isNameEmpty(){
		String name = null;
		if(name==null) {
			//TO DO
		}
		return name.length() == 0;
	}
</pre>

<pre>
	public boolean isNameEmpty() {
		String name = null;
		try {
			return name.length() == 0;
		} catch (NullPointerException e) {
			Log.error(e.getMessage());
			return 0;
		}
	}
</pre>

<p style="color:red">Note: You can throw or catch an Exception instead NullPointerException.</p>