<h1>Forbidden String Equal Expression</h1>
<h2>Noncompliant Code Example</h2>
<pre>
public void hello(String a) {
	a.equals("Your string"); //Please put the constant before the expression in a comparison
}
</pre>
<h2>Compliant Solution</h2>
<pre>
public void hello(String a) {
	"Your string".equals(a);
}
</pre>
<p style="color: red">Note: Put the constant before the expression
	in a comparison.</p>