<h1>Method comment</h1>
<h2>Noncompliant Code Example</h2>
<pre>
	private EventResponse searchSummaryVO(Event e) throws EventException {
		//TO DO
		return eventResponse;
	}
</pre>
<h2>Compliant Solution</h2>
<pre>
	/**
	 * Carry out business scenarios for each event &lt;br&gt;
	 * Branch processing of all events occurring in ALPS-MoneyMgmt system work &lt;br&gt;
	 * 
	 * @param Event e
	 * @return EventResponse
	 * @exception EventException
	 */
	public EventResponse perform(Event e) throws EventException {
		//TO DO
		return eventResponse;
	}
</pre>
<p style="color:red">Note: You can skip the return section if the return is void.</p>