<h1>File Comment</h1>
<h2>Noncompliant Code Example</h2>
<pre>
//Your imports
//Your class comments
public class YourClassName {
	//TO DO
}
</pre>
<h2>Compliant Solution</h2>
<pre>
/*=========================================================
*Copyright(c) 2022 CyberLogitec
*@FileName : YourClassName.java          
*@FileTitle : YourClassName           
*Open Issues : Your issues
*Change history : Your change history
*@LastModifyDate : 2022-08-11
*@LastModifier : Last modifier
*@LastVersion : 1.0
=========================================================*/
//Your imports
//Your class comments
public class YourClassName {
	//TO DO
}
</pre>
<div style="color: red">
	<h2>Note</h2>
	<ul>
		<li>Only content of @FileName and @FileTitle are required.</li>
		<li>Use at most one [Space] on each side of ':'.</li>
		<li>The date format must use the same symbol (./-[Space]) between Year-Month and
			Month-Day. Ex: YYYY-MM-DD; YYYY.MM.DD.</li>
	</ul>
</div>