<h1>Floating Point Values</h1>
<p>Floating-point numbers do not compare as equality (==).
	Floating-point numbers that should be equal are not always equal due to
	rounding problems. Such rounding problems may occur during
	computations.</p>
<h2>Noncompliant Code Example</h2>
<pre>
private void resetAngle(){
    double angle = 0.0;
    ...
    if (angle == 45.5) {
        setUpTime();
    }
}
</pre>

<h2>Compliant Solution</h2>

<pre>
private void calculateAngle(){
    Double angle = new Double(0.0);
    ...
    if (0 == angle.compareTo(new Double(45.5)) {
        setUpTime();
    }
}
</pre>
