<p>This rule detects invalid class comment</p>
<h2>Noncompliant Code Example</h2>
<pre>
	class YourClassName{
		//TO DO
	}
</pre>
<h2>Compliant Solution</h2>
<pre>
	/**
	 * Description for class MyClass_01
	 * 
	 * @author tathienphuoc
	 * @see MyClass_01
	 * @since J2EE 1.6
	 */
	class YourClassName{
		//TO DO
	}
</pre>
